style.css

```diff
@@ -0,0 +1,79 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:ital,wght@1,200&display=swap');
+
```

Comment:
We don't want redundant margins and paddings which come by default to mess with our layout.

For the moment, your display has an unwanted margin around it. We need to remove it:

Your display:
![image](https://user-images.githubusercontent.com/83596135/164491340-2003dfc7-1ecd-4879-9f38-eed4f41b636c.png)


Figma design:
![image](https://user-images.githubusercontent.com/83596135/164491416-c681dbdd-1040-4b11-b16b-4841f1baf0d6.png)

A solution for it will be to place the following and the beginning (in the first lines of code) of your CSS file:
`* {
  margin: 0;
  padding: 0;
  box-sizing: border-box;
}`

This will set the margins and padding to 0 from the beginning; 
And from this format, we start to style elements, overwriting the padding and margins for those who needed it;

It is up to you how you remove the unwanted margins from your project, but we need to do it; :+1: